File: pavan/src/main/java/org/pavan/Address.java
package org.pavan;
class Address {
    private int id;
    private String city;
    private int pincode;

    public Address(int id, String city, int pincode) {
        this.id = id;
        this.city = city;
        this.pincode = pincode;
    }

    @Override
    public String toString() {
        return "Address{" +
                "id=" + id +
                ", city='" + city + '\'' +
                ", pincode=" + pincode +
                '}';
    }
}

File: Vishal/Assignment 1/Address.cs

namespace EmployeeManagement
{
    public class Address
    {
        public int Id { get; set; }
        public string City { get; set; }
        public string Pincode { get; set; }
        public Address(int id,string city,string pincode)
        {
            Id = id;
            City = city;
            Pincode = pincode;
        }
        public override string ToString()
        {
            return $"{City},{Pincode}";
        }
    }
}
